src/main/java/uk/gov/companieshouse/api/strikeoffobjections/processor/ObjectionProcessor.java

```diff
@@ -109,4 +99,34 @@ private void validateObjectionStatus(Objection objection, String httpRequestId)
             throw statusException;
         }
     }
+
+    private void sendInternalEmail(Objection objection, CompanyProfileApi companyProfile,
+                                   String httpRequestId) throws ServiceException {
+        try {
+            emailService.sendObjectionSubmittedDissolutionTeamEmail(companyProfile.getCompanyName(), companyProfile.getJurisdiction(), objection, httpRequestId);
+        } catch (Exception e) {
+            Map<String, Object> logMap = new HashMap<>();
+            logMap.put(LOG_OBJECTION_ID_KEY, objection.getId());
+            apiLogger.errorContext(httpRequestId, "Error sending dissolution team email", e,
+                    logMap);
+
+            throw e;
+        }
+    }
+
+    private void sendExternalEmail(Objection objection, CompanyProfileApi companyProfile,
+                                   String httpRequestId) throws ServiceException {
+        try {
+            emailService.sendObjectionSubmittedCustomerEmail(objection, companyProfile.getCompanyName(), httpRequestId);
+        } catch (Exception e) {
+            Map<String, Object> logMap = new HashMap<>();
+            logMap.put(LOG_OBJECTION_ID_KEY, objection.getId());
+            apiLogger.errorContext(httpRequestId, "Error sending customer email", e, logMap);
+
+            throw e;
+        }
+
+        // TODO update status to processed
```

Comment:
can you move the todo back to the main public method, so it is the last line of the method